packages/dotcom-ui-base-styles/src/lib/documentStyles.ts

```diff
@@ -0,0 +1,10 @@
+import { CSSProperties } from 'react'
+
+export const documentStyles: CSSProperties = {
+  // Enable use of 100vw which does not account for the scroll bar
+  overflowX: 'hidden',
+  // o-colors "paper"
+  backgroundColor: '#fff1e5',
+  // o-colors "black-80"
+  color: '#33302e'
+}
```

Comment:
What is going on in this file?

Reply:
These are the styles to attach to the document element... inlining them isn't strictly necessary but it does enable us to paint a pink background right away.

Reply:
I have updated (and corrected) the documentation to cover this.